src/components/elements/Table/Table.tsx

```diff
@@ -235,9 +251,9 @@ function TableCell<TData extends RowData>({ cell }: { cell: Cell<TData, unknown>
 function LoadingCell() {
   return (
     <tr>
-      <td align="center" colSpan={100} className="py-4 px-6">
+      <td align="center" colSpan={100} className="px-6 py-4">
         <Lottie animationData={SpinnerLottie} className="mb-8 h-8 w-8" />
-        <Text variant="text-light-subtitle-400">Loading results, load times might increase with more data</Text>
+        <Text variant="text-normal-subtitle-400">Loading results, load times might increase with more data</Text>
```

Comment:
Please wrap in `t`

Reply:
Thanks for pointing that out! I've wrapped the text in the t function as recommended